include/boost/stacktrace/detail/mgwhelp_impls.hpp

```diff
@@ -0,0 +1,136 @@
+#ifndef BOOST_STACKTRACE_DETAIL_MGWHELP_IMPLS_HPP
+#define BOOST_STACKTRACE_DETAIL_MGWHELP_IMPLS_HPP
+
+#include <boost/config.hpp>
+#ifdef BOOST_HAS_PRAGMA_ONCE
+#   pragma once
+#endif
+
+#include <boost/stacktrace/frame.hpp>
+#include <boost/detail/winapi/get_current_process.hpp>
+#include <windows.h>
+#include <dbghelp.h>
```

Comment:
I'd like to request two things here:
* some link to the docs or sources of MinGW ensuring that MinGW's dbghelp is thread safe
* some compile time check or `#error` to make sure that MinGW's dbghelp is used